docs/chart-and-series-types/radial-bar-chart.md

```diff
@@ -0,0 +1,40 @@
+Radial Bar (Inverted Polar) Chart
+===
+
+A radial bar (or inverted polar bar) series is a form of visualization column data on polar coordinate system. Since the `xAxis` is vertical and `yAxis` is circular, as opposed to non-inverted variant, shape of points are circular.
+
+<iframe style="width: 100%; height: 600px; border: none;" src=https://www.highcharts.com/samples/embed/highcharts/demo/polar-radial-bar allow="fullscreen"></iframe>
+
+Click [here](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/demo/polar-radial-bar/) to check the code.
+
+Getting started
+---------------
+
+In order to create radial bar chart, the 'highcharts.js' and 'highcharts-more.js' scripts are required to be loaded, just like in case of non-inverted version of a polar chart.
```

Comment:
Also show how we make a column into a radial/circular bar by setting polar and inverted to true.

Reply:
Added the `Creating a radial bar chart` subsection in `Getting started` section and new `Data format` section.